pkg/libs/Core.py

```
@@ -559,19 +559,17 @@ def CopyDependencies(cls):
             var.lib64 + "/ld-linux-x86-64.so*",
             "/lib/ld-musl-x86_64.so*",
         ]
+        result = ""
 
         for libc in libc_paths:
             try:
-                Tools.Info("Attempting libc path: " + libc)
-
                 # Get the interpreter name that is on this system
                 result = check_output("ls " + libc, shell=True, universal_newlines=True).strip()
 
-                if len(result) > 0:
-                    Tools.Info("Libc found")
+                if result:
                     break
                 else:
-                    Tools.Info("Path not found")
+                    result = ""
```

If the interpreter path is not found, this is an error and we should fail right here rather than making it empty and adding it later on (bindeps.add(result)). That just means that the initramfs will be created but will fail at runtime.

Thus, we can invert this check. if not result, then fail. The positive case doesn't need it break since we will continue execution if result was found.